.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Motivation
```

Comment:
@normanmaurer - You don't like the way it looks in UIs, just think it is too noisy as plain text, or something else?